src/components/Home/ClusterDashboardItem.js

```diff
@@ -251,6 +254,22 @@ function ClusterDashboardItem({
               </span>
             </RefreshableLabel>
             <Dot style={{ paddingLeft: 0 }} />
+            <RefreshableLabel value={cluster.kubernetes_version}>
+              <span>
+                <i className='fa fa-kubernetes' title='Kubernetes version' />{' '}
+                <span>
+                  {(() => {
+                    const kubernetes = release.components.find(
+                      (component) => component.name === 'kubernetes'
+                    );
+                    if (kubernetes) return kubernetes.version;
+
+                    return null;
+                  })()}
```

Comment:
The `release` contains the field `kubernetesVersion` which should be used instead of this.

Reply:
Oh. Then I copied a bad example :)